scripts/variantstore/wdl/extract/get_columns_for_import.py

```diff
@@ -243,13 +243,13 @@ def get_column_values(columnSamples, numSamples, user_defined_vcf, user_defined_
         # Check the contents of the columns: the duck algorithm.  If its contents LOOK like vcfs and indexes, go from there
         for col in path_ends_in_vcf_gz:
             # ...and has an analogue that looks like an index file?
-            index_column = f"{column}_index"
+            index_column = f"{col}_index"
```

Comment:
uh what was this doing before these changes?

Reply:
failing!